app/ide-desktop/lib/client/src/log.ts

```diff
@@ -0,0 +1,98 @@
+/** @file Logging utilities. */
+
+import * as fsSync from 'node:fs'
+import * as pathModule from 'node:path'
+
+import * as electron from 'electron'
+
+import * as contentConfig from 'enso-content-config'
+import * as paths from 'paths'
+
+import * as linkedDist from '../../../../../target/ensogl-pack/linked-dist'
+
+const logger = linkedDist.logger
+
+/** Adds a new log consumer that writes to a file.
+ *
+ * The path of the log file is {@link generateUniqueLogFileName automatically generated}.
+ *
+ * The log file is created in the {@link paths.LOGS_DIRECTORY logs directory}
+ *
+ * @returns The full path of the log file.
+ */
+export function addFileLog(): string {
+    const dirname = paths.LOGS_DIRECTORY
+    const filename = generateUniqueLogFileName()
+    const logFilePath = pathModule.join(dirname, filename)
+    const consumer = new FileConsumer(logFilePath)
+    contentConfig.logger.addConsumer(consumer)
+    return logFilePath
+}
+
+/**
+ * Generate a unique log file name based on the current timestamp.
+ *
+ * @returns The file name log file.
+ */
```

Comment:
Our convention is to use comments starting at the same line as `/**`  and `*.` ends on the last line of the comment